README.md

```diff
@@ -47,3 +47,27 @@ Contributions are very welcome, as are feature requests and suggestions. Please
 [codecov-url]: https://codecov.io/gh/JuliaDatabases/ODBC.jl
 
 [issues-url]: https://github.com/JuliaDatabases/ODBC.jl/issues
+
+## Testing
```

Comment:
This is great and I'm sorry it probably took so much effort to figure this out! If you have ideas/suggestions on how to make testing simpler for users vs. travis, I'm all open ears.

Reply:
Nah that's fine, it was pretty straightforward. Setting up easy testing that needs independent dependencies running, for both local testing and on travis, on multiple OSes, seems like a huge can of worms... Travis-first seems like a pragmatic compromise.
I would like to play around a bit with a framework to test on multiple DBs actually (e.g. SQLite), if I ever get time to do that then I'd probably touch test setup at the same time. No promises though sorry :(